### PR TITLE
Add firefox-quick-suggest-weather

### PR DESCRIPTION
**Pull request by htwyford**

### CODE REVIEW

File: taskcluster/ci/config.yml [hunk 1]
```diff
@@ -129,6 +129,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/dynamic-probe-telemetry-extension
             default-ref: main
             type: git
+        quicksuggestweather:
+            name: "Firefox Quick Suggest Weather"
+            project-regex:  firefox-quick-suggest-weather$
+            default-repository: https://github.com/mozilla-extensions/firefox-quick-suggest-weather
```
Comment (escapewindow) [inline taskcluster/ci/config.yml:135]:
```suggestion
            default-repository: git@github.com:mozilla-extensions/firefox-quick-suggest-weather
```

Comment (escapewindow) [reply taskcluster/ci/config.yml:135]: For private repos, we need the ssh format (`git@github.com:org/repo`). That's why the build task is hanging in the PR checks.

**Comment by escapewindow** [comment]
@htwyford It looks like the build is broken. Should `yarn build` work here? https://firefoxci.taskcluster-artifacts.net/Z5Zj_Ki0Rp-D0zKEi1T07g/0/public/logs/live_backing.log

**Comment by htwyford** [comment]
No, I haven't done anything with yarn. I've just been building with `web-ext build` on the command line. What are the next steps here?

**Comment by escapewindow** [comment]
Ah, you need to populate the `scripts` section of your `package.json`. [Here](https://github.com/mozilla-extensions/searchengine-devtools/blob/master/package.json#L22-L30) is an example. The `build` target will need to both create the xpi and name it `web-ext-artifacts/firefox-quick-suggest-weather.xpi`. The easiest way to do this is to use the [`--filename` option in web-ext](https://github.com/mozilla/web-ext/pull/1900). Alternately you can follow searchengine-devtools' example in calling two build targets, the first that creates the xpi, and the 2nd that renames it.

We'll also call the `lint` and `test` scripts.

File: xpi-manifest.yml [hunk 2]
```diff
@@ -203,3 +203,12 @@ xpis:
...
     addon-type: privileged
     install-type: npm
+  - name:  firefox-quick-suggest-weather
+    description: Firefox Quick Suggest Weather
+    repo-prefix: quicksuggestweather
+    active: true
+    private-repo: true
+    artifacts:
+      - web-ext-artifacts/firefox-quick-suggest-weather.xpi
+    addon-type: privileged
```
Comment (escapewindow) [inline xpi-manifest.yml:213]:
```suggestion
    addon-type: normandy-privileged
```

Comment (escapewindow) [reply xpi-manifest.yml:213]: I think you mentioned this was going to be a Normandy study, correct?

**Review by escapewindow — APPROVED** [review]
It looks like you already [successfully set up CI](https://github.com/mozilla-extensions/firefox-quick-suggest-weather/runs/1537817380); thank you! I reran the task here, green.

**Pull request merged**